examples/src/MenuStructure.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+
+import { LoginFormAnt } from './form/LoginFormAnt';
+import { MemTableAnt } from './memtable/MemTableAnt';
+import { ApplicationAnt } from './app/ApplicationAnt';
+
+import lockImgUrl from "../images/lock.jpg";
+
+import {
+    //    UrlArg,
+    //    URLARG_TYPE_STRING,
+    StateSpace,
+} from '@moxb/moxb';
+
+export const mainMenu: StateSpace = [
+    {
+        path: 'index',
+        root: true,
+        label: 'All Components',
+        fragment: <ApplicationAnt />,
+    },
+    {
+        path: 'loginForm',
+        label: <span><img src={ lockImgUrl } width="32"/>Login Form</span>,
```

Comment:
`label` can be either a string, or some HTML content, or even a react component, or a component class.